packages/react-aria-components/docs/ListBox.mdx

```diff
@@ -568,6 +568,239 @@ import {Text} from 'react-aria-components';
 </ListBox>
 ```
 
+### Horizontal layout
+
+By default, ListBox expects items to be arranged in a vertical stack, and implements keyboard navigation and drag and drop accordingly. The `layout` and `orientation` props can be used to change this behavior, allowing you to build horizontal stack, grid, and column layouts.
```

Comment:
row, column, and grid layouts?
seems weird to say horizontal stack but not vertical stack, and CSS flex refers to them as row and column, so I think that's natural to say. where as stacks are really from iOS and a few other places that aren't web

Reply:
i've been going back and forth while reviewing, because I do like `layout: stack | grid` because you could implement the grid using flex, so `layout: flex | grid` doesn't make sense, so I can understand using horizontal stack as well, but I still think we should choose either `horizontal/vertical stack and grid` or `stack and grid` or `row, column, and grid`

Reply:
yeah `flex` doesn't really describe how the items are arranged. I said vertical stack in the previous sentence when describing the default so I didn't think I needed to repeat that?